.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -90,37 +82,8 @@ set CXX=sccache-cl
 
 set CMAKE_GENERATOR=Ninja
 
-:: The following code will try to build PyTorch twice if USE_CUDA is neither 0
-:: nor 1. It is intended so that both builds can be folded into 1 CI run.
```

Comment:
Just to confirm, we no longer try to build twice in one run now?

Reply:
Yeah, we stopped to do that after migrating to circleci.